ddtrace/internal/flare.py

```diff
@@ -29,13 +28,21 @@
 log = get_logger(__name__)
 
 
+@dataclasses.dataclass
+class FlareSendRequest:
+    case_id: str
+    hostname: str
+    email: str
+    source: str = "tracer_python"
+
+
 class Flare:
     def __init__(self, timeout_sec: int = DEFAULT_TIMEOUT_SECONDS):
-        self.original_log_level = 0  # NOTSET
+        self.original_log_level = logging.NOTSET
         self.timeout = timeout_sec
         self.file_handler: Optional[RotatingFileHandler] = None
 
-    def prepare(self, configs: List[dict]):
+    def prepare(self, log_level: str):
```

Comment:
I forgot, we could help with typing here too:

```python
LogLevelString = typing.Literal["DEBUG"] | typing.Literal["WARNING"] | ...

def prepare(self, log_level: LogLevelString):
    pass
```


this is just a 💡, you don't need to incorporate. since this is a private api, we can get away with the slightly looser typing.